quic/s2n-quic-qns/src/perf.rs

```diff
@@ -278,8 +305,12 @@ impl Counters {
         let duration = self.timeout.swap(0, Ordering::Relaxed);
         let duration = Duration::from_nanos(duration);
         let pto_count = self.pto_count.swap(0, Ordering::Relaxed);
+        let max_pacing_rate = self.max_pacing_rate.swap(0, Ordering::Relaxed);
+        let max_pacing_rate = rate(max_pacing_rate, Duration::from_secs(1));
+        let max_delivery_rate = self.max_delivery_rate.swap(0, Ordering::Relaxed);
+        let max_delivery_rate = rate(max_delivery_rate, Duration::from_secs(1));
         eprintln!(
-            "{send_rate}\t{receive_rate}\t{max_cwnd}\t{max_bytes_in_flight}\t{lost_packets}\t{wakeups}\t{duration:?}\t{max_rtt:?}\t{max_smoothed_rtt:?}\t{pto_count}",
+            "{send_rate}\t{receive_rate}\t{max_cwnd}\t{max_bytes_in_flight}\t{lost_packets}\t{wakeups}\t{duration:?}\t{max_rtt:?}\t{max_smoothed_rtt:?}\t{pto_count}\t{max_pacing_rate}\t{max_delivery_rate}",
```

Comment:
As time goes its harder to remember what all these mean. We should do it for others also but can you add a comment explaining what these two new fields mean and how to interpret them.

Your comment about Cubic vs BBR for example.

Reply:
Probably worth adding a header to the output, too. Nice thing is you can pipe it into a file and you have a tsv

Reply:
I went ahead and added comments for all of them

Reply:
whoops that format isn't going to work :-) I'll fix that and add the header